src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import "./App.css";
+
+const Form = props => {
+    console.log(props);
+    const [team, setTeam] = useState({ name: "", email: "", role:""});
+
+    const changeHandler = event => {
+        setTeam({...team, [event.target.name]: event.target.value});
+    };
+
+    const submitForm = event => {
+        event.preventDefault();
+        setTeam({name:"", email:"", role:""});
```

Comment:
This looks like it's resetting the form state here. You want to make sure you pass down a prop so you can add the **team** variable data do the **teams** array you're storing at the top level (inside _index.js_)